### PR TITLE
Added android app link that opens the app from the email confirmation

### PR DESCRIPTION
**Pull request by Lyton505**
## Describe your changes
- Implemented opening the app from the confirmation URL sent by supabase

## Issue ticket number and link
#20 

## Screenshots / Video

https://github.com/Firm-Collective/ai-journal/assets/65817176/f5a4a53c-524d-49dd-9555-6e9720d848ea

## Checklist
- [x] I have performed a self-review of my code
- [x] I have used `npx expo-doctor` to review any dependency issues.
- [x] I have pulled and merged any recent changes from the main branch (no merge conflicts)
- [x] I have tested thoroughly any edge cases that might break the application
- [x] I have deleted any console.log statements and useless comments

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -38,14 +59,13 @@
...
-    "runtimeVersion": {
-      "policy": "appVersion"
-    },
+    "runtimeVersion": "1.0.0",
     "updates": {
-      "url": "https://u.expo.dev/ca2a4644-0c86-4092-b1f9-d554abfc52cf"
-    }
+      "url": "https://u.expo.dev/3196a892-2eac-48b2-89b1-69cb37e04986"
+    },
+    "owner": "lyton505"
```
Comment (dave-cao) [inline app.json:69]: I will remove this line. In the next meeting I will ask Lewis about making a client eas account.

File: app.json [hunk 2]
```diff
@@ -38,14 +59,13 @@
         "origin": false
       },
       "eas": {
-        "projectId": "ca2a4644-0c86-4092-b1f9-d554abfc52cf"
+        "projectId": "3196a892-2eac-48b2-89b1-69cb37e04986"
```
Comment (dave-cao) [inline app.json:62]: I will revert this for now.

File: lib/Auth.ts
Comment (dave-cao) [inline lib/Auth.ts:1]: Lines 2-5 are not used within this file. Will delete.

**Review by dave-cao — APPROVED** [review]
Made a few changes, but will merge this. Looks good and works on my end!

- removed "owner" in app.json
- reverted "projectId" and "url" in app.json
- removed unused imports in /lib/Auth.ts

**Comment by dave-cao** [comment]
image

I am getting a secureStore error saying it's over 2048 bytes. 

This only occurs for users who have confirmed their email. My `test@email.com` account which didn't confirm the email didn't have this error. Not sure if we should look for an alternative storage method or figure out a way to reduce the size.

**Comment by dave-cao** [comment]
> image
> 
> I am getting a secureStore error saying it's over 2048 bytes.
> 
> This only occurs for users who have confirmed their email. My `test@email.com` account which didn't confirm the email didn't have this error. Not sure if we should look for an alternative storage method or figure out a way to reduce the size.

I updated the secureStorage to compress the data before storing. I will push the changes now. If you can test it to see if it still works on your end, I will merge the changes :)

**Comment by Lyton505** [comment]
I hadn't tested a non existent email. Good catch. Seems to work on my end now

**Comment by dave-cao** [comment]
> I hadn't tested a non existent email. Good catch. Seems to work on my end now

Sounds good, I'll merge it now!

**Pull request merged**